Orange/widgets/visualize/tests/test_owsieve.py

```diff
@@ -46,3 +51,26 @@ def test_keyerror(self):
         data = Table("iris")
         data = data[0:1]
         self.send_signal("Data", data)
+
+    def test_chisquare(self):
+        """
+         gh-2031
+         Check if it can calculate chi square when there are no attributes which suppose to be.
+        """
+        tempdir = tempfile.mkdtemp()
```

Comment:
Instead of creating files in code, you could either commit the file along the test, or even better create the table directly with something along the lines of:

```
a, b = Orange.data.DiscreteVariable("a", values=["y", "n"]), Orange.data.DiscreteVariable("b", values=["y", "n", "other"])
t = Orange.data.Table(Orange.data.Domain([a, b], list(zip("yynny", "ynyyn"))))
```